### PR TITLE
Catch error in If statement. Also, newline for EOF

### PR DESCRIPTION
**Pull request by Beasta**

### CODE REVIEW

File: scripts/mint.js [hunk 1]
```diff
@@ -78,7 +78,9 @@ async function main() {
             const result = await factoryContract.methods.mint(LOOTBOX_OPTION_ID, OWNER_ADDRESS).send({ from: OWNER_ADDRESS });
             console.log("Minted lootbox. Transaction: " + result.transactionHash)
         }
+    } else {
+      console.error('Add NFT_CONTRACT_ADDRESS or FACTORY_CONTRACT_ADDRESS to the environmental variables')
```
Comment (alexanderatallah) [inline scripts/mint.js:82]: "to your environment variables" would be better here

**Comment by Beasta** [comment]
PR has been updated

File: scripts/mint.js [hunk 2]
```diff
@@ -79,7 +79,7 @@ async function main() {
             console.log("Minted lootbox. Transaction: " + result.transactionHash)
         }
     } else {
-      console.error('Add NFT_CONTRACT_ADDRESS or FACTORY_CONTRACT_ADDRESS to the environmental variables')
+      console.error('Add NFT_CONTRACT_ADDRESS or FACTORY_CONTRACT_ADDRESS to your environmental variables')
```
Comment (alexanderatallah) [inline scripts/mint.js:82]: I also meant changing "environmental" to "environment", just to be consistent with what most people say. Sorry to be pedantic. Approving preemptively

Comment (Beasta) [reply scripts/mint.js]: No problem. Can't believe I missed it. 🤦‍♂️

**Comment by Beasta** [comment]
i think i got it finally :)

**Pull request closed without merging**